src/Materials/fresnelParameters.ts

```diff
@@ -57,37 +128,47 @@ export class FresnelParameters {
         return newFresnelParameters;
     }
 
+    /**
+     * Determines equality between FresnelParameters objects
+     * @param otherFresnelParameters defines the second operand
+     * @returns true if the power, bias, leftColor, rightColor and isEnabled values are equal to the given ones
+     */
+    public equals(otherFresnelParameters: DeepImmutable<FresnelParameters>): boolean {

```

Comment:
If it is only for tests, should we introduce it here ?

Reply:
I added `equals(...)` as I use it for diffing out changes when I reconcile changes and need to update the BabylonJS scene.  If you think they are not useful then I can remove - I am okay to check all properties individually when I diff.